File: Zadaca_5.2/Brand.cs
using System;
using System.Collections.Generic;

namespace MVC
{
    public partial class Brand
    {
        public byte BrandId { get; set; }
        public string? BrandName { get; set; }
        public byte? BrandActive { get; set; }
        public byte? BrandStatus { get; set; }
    }
}

File: Zadaca_6/graphQL-express-MySQL-example-main/types/Brand.js
const { GraphQLObjectType, GraphQLInt, GraphQLString, GraphQLList } = require('graphql');
const { dbQuery } = require('../database');

const BrandType = new GraphQLObjectType({
  name: 'Brand',
  fields:() => (
    {
      brand_id: { type: GraphQLInt },
      brand_name: { type: GraphQLString },
      brand_active: { type: GraphQLInt },
      brand_status: { type: GraphQLInt }
    }
  ) 
});

module.exports = BrandType;